docs/crud/property/schema.md

```diff
@@ -0,0 +1,124 @@
+# CRUD Property
+
+CRUD operations create/update, read and delete property.
+
+Property stores the user defined data.
+
+[`bydbctl`](../../clients.md#command-line) is the command line tool in examples.
+
+## Apply (Create/Update) operation
+
+Apply creates a property if it's absent, or updates an existed one based on a strategy. If the property does not currently exist, create operation will create the property.
+
+### Examples of applying
+
+A property belongs to a unique group. We should create such a group before creating a property.
+
+The group's catalog should be empty.
+
+```shell
+$ bydbctl group create -f - <<EOF
+metadata:
+  name: default
+schema_opts:
+  shard_num: 2
+  block_interval:
+    unit: UNIT_HOUR
+    num: 2
+  segment_interval:
+    unit: UNIT_DAY
+    num: 1
+  ttl:
+    unit: UNIT_DAY
+    num: 7
```

Comment:
In the UI template use case, the TTL should not be set. Could you be more specific that this is optional?

Reply:
The `ResourceOpt` should be left empty here. I also remove `requried` validation from APIs.